CHANGES.rst

```diff
@@ -162,6 +162,10 @@ astropy.units
 astropy.utils
 ^^^^^^^^^^^^^
 
+- Leap seconds are now automatically inferred from IERS tables, ensuring that
+  in normal usage they will not be out of date even if astropy and erfa are
```

Comment:
When does "normal usage" check for new versions of the leap seconds table? When it's been six months since the last bulletin C? When a time is encountered that is more than six months since the last Bulletin C? When someone needs IERS A data for some other non-leap-second purpose?